src/linpred.jl

```diff
@@ -113,14 +128,44 @@ end
 Base.cholfact{T}(p::SparsePredChol{T}) = copy(p.chol)
 Base.cholfact!{T}(p::SparsePredChol{T}) = p.chol
 
+"""
+    coef(x)
+
+Extract the estimates of the coefficients in the model.
+
+#### Arguments:
+* `x`: Inputs can either `LinPred` or `LinPredModel`.
+
+#### Examples:
+Refer example for `glm()`
```

Comment:
A partial example isn't terribly practical. Probably remove it completely, as this is such basic functionality that it is covered elsewhere.